crates/ruff/src/jupyter/snapshots/ruff__jupyter__notebook__tests__import_sorting.snap

```diff
@@ -47,4 +47,45 @@ isort.ipynb:cell 2:1:1: I001 [*] Import block is un-sorted or un-formatted
 7 9 | def foo():
 8 10 |     pass
 
+isort.ipynb:cell 2:6:1: I001 [*] Import block is un-sorted or un-formatted
+   |
+ 4 |   def foo():
+ 5 |       pass
+ 6 | / from pathlib import Path
+ 7 | | import sys
+ 8 | | 
+ 9 | | %matplotlib \
+   | |_^ I001
+10 |       --inline
+   |
+   = help: Organize imports
```

Comment:
This is incorrect as line 4 and 5 belong to the previous cell, so it shouldn't be shown here. This is more of a todo for me. It shows the correct information when using `cargo run` on the file directly. It seems the the Jupyter index is computed when it shouldn't be, thus the incorrect cell breaks.

Reply:
fixed in [`b122d14` (#5552)](https://github.com/astral-sh/ruff/pull/5552/commits/b122d14a5f636812ef54151c10acb1be11c8e560)